### PR TITLE
carbon/alien retype and associated carbon removals.

### PR DESCRIPTION
**Pull request by MistakeNot4892**
## Description of changes
- `/mob/living/carbon/alien` has been retyped to `/mob/living/simple_animal/alien`. This is a pretty coarse conversion and I expect some stuff will be broken, but I would like to get the type rework in then fix issues as they arrive, as there will be a lot more issues from the remaining carbon removals.
- Removes `iscarbon()` and all uses, retyping things to either `/mob/living` or `/mob/living/carbon/human` as appropriate.

## Why and what will this PR improve
Carbon delenda est.

## Authorship
Myself.

## Changelog
In theory, nothing player-facing.

### CODE REVIEW

**Comment by MistakeNot4892** [comment]
Going to split this one up into several PRs I think.

**Comment by MistakeNot4892** [comment]
Depends on #3690 #3693

File: code/game/gamemodes/cult/runes.dm [hunk 1]
```diff
@@ -681,13 +681,12 @@
 		if(N)
 			continue
 		affected |= M
-		if(iscarbon(M))
-			var/mob/living/carbon/C = M
-			SET_STATUS_MAX(C, STAT_BLURRY, 50)
-			SET_STATUS_MAX(C, STAT_WEAK, 3)
-			SET_STATUS_MAX(C, STAT_STUN, 5)
-		else if(issilicon(M))
+		if(M.isSynthetic())
```
Comment (out-of-phaze) [inline code/game/gamemodes/cult/runes.dm]: This is maybe an unintentional buff to FBPs, though it's not like confuse runes actually get used all that often.

File: code/modules/mob/living/carbon/human/human.dm [hunk 2]
```diff
@@ -1173,6 +1173,13 @@
 		return SScharacter_info.get_record(comments_record_id, TRUE)
 	return ..()
 
+/mob/living/carbon/human/proc/get_age()
+	. = LAZYACCESS(appearance_descriptors, "age") || 30
+
+/mob/living/carbon/human/proc/set_age(var/val)
+	var/datum/appearance_descriptor/age = LAZYACCESS(species.appearance_descriptors, "age")
+	LAZYSET(appearance_descriptors, "age", age.sanitize_value(val))
```
Comment (out-of-phaze) [inline code/modules/mob/living/carbon/human/human.dm]: We use LAZYACCESS on the line before, but `age` isn't nullchecked here.

**Review by out-of-phaze — CHANGES_REQUESTED** [review]

**Review by out-of-phaze — DISMISSED** [review]

**Review by out-of-phaze — APPROVED** [review]

**Pull request merged**